data/shared/citizen/scripting/lua/scheduler.lua

```diff
@@ -472,7 +472,7 @@ local function doStackFormat(err)
 	local fst = FormatStackTrace()
 	
 	-- already recovering from an error
-	if not fst then
+	if not fst or not err then
```

Comment:
This will silently suppress a `nil` error, and won't fix the case where an error object is e.g. a `number` or `boolean`. Doing something like the Lua codebase itself does might be better:

https://github.com/lua/lua/blob/e288c5a91883793d14ed9e9d93464f6ee0b08915/lua.c#L135-L142

Reply:
At the very least `string.format` would resolve the issue with concatenating non-string values.
```lua
return string.format('^1SCRIPT ERROR: %s^7\n%s', err or '', fst)
```